src/components/GamePage.tsx

```
@@ -6,25 +6,56 @@ import { handleMove } from '../api/deepseek';
-  const [gameOverMessage, setGameOverMessage] = useState<string | null>(null);
+  const [alert, setAlert] = useState(null);
+  const [moveHistory, setMoveHistory] = useState([]);
+  const [gameOverMessage, setGameOverMessage] = useState(null);
   const [boardWidth, setBoardWidth] = useState(600);
-  const [aiLoading, setAiLoading] = useState<boolean>(false);
-  const [leaderboard, setLeaderboard] = useState<{ name: string; moves: number }[]>([]);
-  const [matchStatus, setMatchStatus] = useState<string>('Match is Live');
-  const [isFirstMove, setFirstMove] = useState<boolean>(true);
-  const playerName = localStorage.getItem('playerName') || '';
+  const [aiLoading, setAiLoading] = useState(false);
+  const [leaderboard, setLeaderboard] = useState([]);
```

ESLint

[error] 'leaderboard' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)